app/src/main/java/com/example/sharingang/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.example.sharingang
 
-import androidx.appcompat.app.AppCompatActivity
+import android.content.Intent
 import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil
+import com.example.sharingang.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        val binding: ActivityMainBinding =
+            DataBindingUtil.setContentView(this, R.layout.activity_main)
```

Comment:
Why is "binding" just a local variable, whereas in NewItemActivity it is an attribute?

Reply:
In `NewItemActivity`, it needs to be an attribute, since we need to reference it later in the class. We use the description when creating our Intent. Here, we don't use it anywhere else in the activity, so it's better to just keep things as a local variable to this method.